tests/Test.hx

```diff
@@ -0,0 +1,4 @@
+class Test {
```

Comment:
Is this dummy test going to be merged down as well? If so, why is it needed?

Reply:
Yes it needs to be merged because it's needed for the `travis.yml` build, right now we don't have anything meaningful to test yet.